package.json

```diff
@@ -33,7 +33,7 @@
     },
     {
       "path": "static/build/all.js",
-      "maxSize": "144.3KB"
+      "maxSize": "145.77KB"
```

Comment:
Round up to 145.8KB for consistency with the other numbers and a bit of "wiggle room" :)

Reply:
Wiggle room is good.  Change has been made in most recent commit.